shipment_advice_planner/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+#. Go to Inventory>Operations>transfers.
+#. Select transfers to plan.
+#. Click on "Plan shipments".
```

Comment:
This action will likely launch the shipment advice wizard of the shipment_advice module.
I don't understand how the ShipmentAdvicePlanner is launched.

Reply:
the wizard is the planner,